realsense2_camera/include/base_realsense_node.h

```diff
@@ -6,423 +6,429 @@
 #define ___BASE_REALSENSE_NODE_HEADER___
```

Comment:
You re-styled the whole code. Please don't do that, since we will loose track of the changes that we have done with respect to the upstream. Also, fetching updates from the upstream will give us a lot of merge conflicts and will be a pain. Finally, with this restyle I don't know which changes am I supposed to review